docs/gettingstarted/communitystds.md

```diff
@@ -1,11 +1,11 @@
 # Community Standards
 
-We view this as an obligation to all of our Members. Any Members violating 
-the Community Standards will be removed from the community. Violations should 
-be reported to [support@chingu.io](mailto:support@chingu.io) and will be kept 
+We view this as an obligation to all of our Members. Any Members violating
+the Community Standards will be removed from the community. Violations should
```

Comment:
not the ticket system?

Reply:
Agreed. Changed to direct them to open a support ticket.